src/scenario.jl

```diff
@@ -695,7 +714,7 @@ function run_model(domain::Domain, param_set::NamedDimsArray, corals::DataFrame,
         end
 
         # Fog selected locations
-        if (fogging > 0.0) && in_shade_years && has_shade_sites
+        if (fogging > 0.0) && in_fog_years && has_fog_sites
```

Comment:
If `apply_fogging = fogging > 0.0` is adopted:

```suggestion
        if apply_fogging && in_fog_years && has_fog_sites
```